### PR TITLE
[wasm] make debugger less noisy

### PR DESCRIPTION
**Pull request by pavelsavara**
Because it matters in shell or NodeJS https://github.com/dotnet/runtime/issues/63703

### CODE REVIEW

File: src/mono/wasm/runtime/debug.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export function mono_wasm_runtime_ready(): void {
         // eslint-disable-next-line no-debugger
         debugger;
     else
-        console.debug("mono_wasm_runtime_ready", "fe00e07a-5519-4dfe-b35a-f867dbaf2e28");
+        console.assert(true, "mono_wasm_runtime_ready", "fe00e07a-5519-4dfe-b35a-f867dbaf2e28");
```
Comment (lewing) [inline src/mono/wasm/runtime/debug.ts:48]: is assert available everywhere?

Comment (thaystg) [reply src/mono/wasm/runtime/debug.ts:48]: It does not work on firefox, trying to find another solution.

Comment (pavelsavara) [reply src/mono/wasm/runtime/debug.ts:48]: The browser API is everywhere. How it's reported to debugger protocol is different for FF.
https://caniuse.com/mdn-api_console_assert

**Review by radekdoulik — APPROVED** [review]
LGTM

I wonder about the `src/mono/wasm/runtime/package-lock.json` file. Do we need it in the repo?

**Comment by pavelsavara** [comment]
> LGTM
> 
> I wonder about the `src/mono/wasm/runtime/package-lock.json` file. Do we need it in the repo?

Yes, we do need it. The problem is that emsdk has older version of nodeJS than most of us have on the system. 
When I did package upgrade last week, we I used node on my system, which generated lock file version 2 (backward compatible with v1). But when we run the same command as part of build, it would run npm with smsdk node version and that would produce the lock file in v1 format and create this delta. I guess it's better to commit the delta and stick with v1 lock file, than to fight it on each commit.

File: src/mono/wasm/runtime/debug.ts (hunk 1 above)
Comment (lambdageek) [inline src/mono/wasm/runtime/debug.ts:48]: Is this some node trick?

**Comment by pavelsavara** [comment]
Closing in favor of https://github.com/dotnet/runtime/pull/72812

**Pull request closed without merging**